tests/example_pytest.ini

```diff
@@ -0,0 +1,16 @@
+[pytest]
```

Comment:
Let's not conflate multiple tests into one, and please move the conf closer to the test as shown in the current test example.